website/routes.py

```diff
@@ -1462,6 +1462,16 @@ def make_url_map(app):
         ),
 
         Rule(
+            [
+                '/project/<pid>/get_node_tree/',
```

Comment:
Avoid verbs in routes; that's what the HTTP verbs are for. How about 

```
/project/<pid>/tree/
```

Need @brianjgeiger to sign off on this one.